plugins/kms/const.go

```diff
@@ -1,3 +1,12 @@
 package kms_plugin_assets
 
-const KmsPluginPrefix = "boundary-plugin-kms-"
+import (
+	"runtime"
+	"strings"
+)
+
+const kmsPluginPrefix = "boundary-plugin-kms"
+
+func KmsPluginPrefix() string {
```

Comment:
Rather than doing this, you could also just change the build script to put each plugin in a separate directory.

This would also get rid of the need for globbing in the embed line.

Reply:
Also it might be cleaner to leave this as a const defined in the individual files as opposed to a function 🤷 It doesn't super matter and we can change it later...